lib/rules/define-macros-order.js

```diff
@@ -130,6 +130,18 @@ function create(context) {
       },
       onDefineExposeExit(node) {
         defineExposeNode = getDefineMacrosStatement(node)
+      },
+
+      'CallExpression:exit'(node) {
+        // check if the node is a macro in the order
```

Comment:
I think this comment is not necessary, the condition is quite readable on its own.

```suggestion
```